opts/opts.go

```diff
@@ -22,6 +22,8 @@ type Opts struct {
 	Version int
 	Security
 	HostCfg
+	SigningRoot
+	HttpsRoots
```

Comment:
What about
```
SigningRoot *x509.Certificate
HttpsRoots []*x509.Certificate
```
Help me, if I miss something here, pls.

Reply:
That was leftover from a different approach. Must have slipped